Public/Get-SeElement.ps1

```diff
@@ -105,10 +105,14 @@ function Get-SeElement {
             }
               
         }
-        
-        #Issue #135 - Explicit $null so it get picked up by downstream cmdlets
-        if ($null -eq $Output) { return $null } else { return $Output }
-        
+        if ($null -eq $Output) {
+            $Message = "no such element: Unable to locate element by: $($By -join ',') with value $($Value -join ',')"
+            Write-Error -Exception ([System.InvalidOperationException]::new($Message))
```

Comment:
What do you think about using `[System.Management.Automation.ItemNotFoundException]` instead?

IMO `InvalidOperation` is generally used for things that would never work and can communicate that user input is somehow completely unusable, which doesn't feel like is the case here. 🙂

Reply:
@vexx32 Yes, I like yours better. 

I used InvalidOperation because that's what the native call produced and didn't have ItemNotFound exception in mind.

Making the change right now.